app/controllers/users.js

```diff
@@ -0,0 +1,31 @@
+const userModel = require('../models').user,
+  usersService = require('../services/users'),
+  logger = require('../logger'),
+  bcrypt = require('bcryptjs');
+
+const encryptPassword = password => bcrypt.hashSync(password, bcrypt.genSaltSync());
+
+exports.create = (req, res) =>
+  userModel
+    .findOne({ where: { email: req.body.email } })
+    .then(user => {
+      if (user) {
+        logger.info(`The user ${user} already exists`);
+        res.status(422).send(`The user with email: ${user.email} already exists`);
```

Comment:
Maybe 400? In this case, the email does not correct.

Reply:
Yeah... mi doubt was about using `400` or `422`. I think that `400` es better.